src/Ledger/Conway/Specification/Gov/Actions.lagda

```diff
@@ -225,6 +246,26 @@ instance
   unquoteDecl DecEq-Vote          = derive-DecEq ((quote Vote    , DecEq-Vote)    ∷ [])
   unquoteDecl DecEq-VDeleg        = derive-DecEq ((quote VDeleg  , DecEq-VDeleg)  ∷ [])
 
+  DecEq-GovVoter : DecEq GovVoter
```

Comment:
Can this not be derived?

Reply:
It seems derive-DecEq does not support dependent-pair like record types